stl/inc/numbers

```diff
@@ -0,0 +1,98 @@
+    inline constexpr _Floating pi_v = static_cast<_Floating>(3.14159265358979323846);
+    template <floating_point _Floating>
+    inline constexpr _Floating inv_pi_v = static_cast<_Floating>(0.318309886183790671538);
+    template <floating_point _Floating>
+    inline constexpr _Floating inv_sqrtpi_v = static_cast<_Floating>(0.564189583547756286948);
+    template <floating_point _Floating>
+    inline constexpr _Floating ln2_v = static_cast<_Floating>(0.693147180559945309417);
+    template <floating_point _Floating>
+    inline constexpr _Floating ln10_v = static_cast<_Floating>(2.30258509299404568402);
+    template <floating_point _Floating>
+    inline constexpr _Floating sqrt2_v = static_cast<_Floating>(1.41421356237309504880);
+    template <floating_point _Floating>
+    inline constexpr _Floating sqrt3_v = static_cast<_Floating>(1.73205080756887729353);
+    template <floating_point _Floating>
+    inline constexpr _Floating inv_sqrt3_v = static_cast<_Floating>(0.577350269189625764509);
+    template <floating_point _Floating>
+    inline constexpr _Floating egamma_v = static_cast<_Floating>(0.577215664901532860607);
+    template <floating_point _Floating>
+    inline constexpr _Floating phi_v   = static_cast<_Floating>(1.61803398874989484820);
+#else // ^^^ defined(__cpp_lib_concepts) / !defined(__cpp_lib_concepts) vvv
+    template <class _Ty>
+    using _Floating = enable_if_t<is_floating_point_v<_Ty>, _Ty>;
+
+    template <class _Ty>
+    inline constexpr _Ty e_v = static_cast<_Floating<_Ty>>(2.71828182845904523536);
+    template <class _Ty>
+    inline constexpr _Ty log2e_v = static_cast<_Floating<_Ty>>(1.44269504088896340736);
+    template <class _Ty>
+    inline constexpr _Ty log10e_v = static_cast<_Floating<_Ty>>(0.434294481903251827651);
+    template <class _Ty>
+    inline constexpr _Ty pi_v = static_cast<_Floating<_Ty>>(3.14159265358979323846);
+    template <class _Ty>
+    inline constexpr _Ty inv_pi_v = static_cast<_Floating<_Ty>>(0.318309886183790671538);
+    template <class _Ty>
+    inline constexpr _Ty inv_sqrtpi_v = static_cast<_Floating<_Ty>>(0.564189583547756286948);
+    template <class _Ty>
+    inline constexpr _Ty ln2_v = static_cast<_Floating<_Ty>>(0.693147180559945309417);
+    template <class _Ty>
+    inline constexpr _Ty ln10_v = static_cast<_Floating<_Ty>>(2.30258509299404568402);
+    template <class _Ty>
+    inline constexpr _Ty sqrt2_v = static_cast<_Floating<_Ty>>(1.41421356237309504880);
+    template <class _Ty>
+    inline constexpr _Ty sqrt3_v = static_cast<_Floating<_Ty>>(1.73205080756887729353);
+    template <class _Ty>
+    inline constexpr _Ty inv_sqrt3_v = static_cast<_Floating<_Ty>>(0.577350269189625764509);
+    template <class _Ty>
+    inline constexpr _Ty egamma_v = static_cast<_Floating<_Ty>>(0.577215664901532860607);
+    template <class _Ty>
+    inline constexpr _Ty phi_v = static_cast<_Floating<_Ty>>(1.61803398874989484820);
+#endif

```

Comment:
`#endif` comment again. (Here is a good example of being able to forget what region we're in.)